tests/opwen_email_server/mailers/test_wikipedia.py

```diff
@@ -0,0 +1,41 @@
+from unittest import TestCase
+
+import responses
+
+from opwen_email_server.mailers.wikipedia import WikipediaEmailFormatter
+
+
+class WikipediaServiceTests(TestCase):
+    def setUp(self):
+        self._formatter = WikipediaEmailFormatter()
+
+    def test_no_results(self):
+        email = {
+            'to': ['wikipedia@bot.lokole.ca'], 'from': 'user@lokole.ca', 'subject': 'en', 'body': 'arandomsearch',
+            'sent_at': '2020-02-01 21:17'
+        }
+        result_email = self._formatter(email)
+        self.assertTrue(result_email['subject'] == 'No results')
```

Comment:
Nit: you can simplify this and also get a better error message on failure via:

```suggestion
        self.assertEqual(result_email['subject'], 'No results')
```